src/Symfony/Component/Form/ResolvedFormType.php

```diff
@@ -56,9 +56,7 @@ public function __construct(FormTypeInterface $innerType, array $typeExtensions
     }
 
     /**
-     * Returns the prefix of the template block name for this type.
-     *
-     * @return string The prefix of the template block name
+     * {@inheritdoc}
```

Comment:
@webmozart why not simply call `$type->getInnerType()->getBlockPrefix()` there? It seems strange to me to offer two ways to do the same thing.